thinkhazard/views/admin.py

```diff
@@ -41,12 +39,39 @@
     HazardSet,
     Layer,
     TechnicalRecommendation,
+    Publication,
 )
+import thinkhazard.celery as tasks
 
+TASKS_LABELS = {
+    "publish": "Publication",
+    "transifex_fetch": "Import from Transifex",
+    "admindivs": "Import administrative division from GeoNode",
+    "process": "Processing",
```

Comment:
Processing => Harvest and process Geonode datasets (start with verb to be consitent)